site/src/examples/date-picker/Range.tsx

```diff
@@ -0,0 +1,6 @@
+import { ReactElement } from "react";
+import { DatePicker } from "@salt-ds/lab";
+
+export const Range = (): ReactElement => (
+  <DatePicker selectionVariant="range" style={{ width: "250px" }} />
```

Comment:
This example's year dropdown is truncated?

Reply:
i think that is a problem with how the site is rendering fonts, it happens in calendar too